modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -262,12 +270,14 @@ namespace {{packageName}}.Client
         /// <param name="name">Parameter name.</param>
         /// <param name="stream">Input stream.</param>
         /// <returns>FileParameter.</returns>
-        public FileParameter ParameterToFile(string name, Stream stream)
-        {
-            if (stream is FileStream)
+        public FileParameter ParameterToFile(string name, Stream stream) {
+            log.Trace("start");
+            if (stream is FileStream) {
+                log.Debug("FileStream");
```

Comment:
But you have a log time usually which allows you to sort out all of this mess and get the proper order, don't you?